core/data/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/data/announcement/AnnouncementsApi.kt

```diff
@@ -18,7 +18,7 @@ public class AnnouncementsApi(
         language: String
     ): PersistentList<AnnouncementsByDate> {
         return networkService.get<AnnouncementsResponse>(
-            url = "${BuildKonfig.apiUrl}/events/droidkaigi2022/announcements/$language"
+            url = "${BuildKonfig.apiUrl}/announcements/$language"
```

Comment:
The URL was set to the same URL as other APIs, but this was corrected because it resulted in a 404 error.